# change to motor index instead? 
int8 motor_index 0
int8 motor_move 0
bool exit false